kadi/scripts/validate_states.py

```diff
@@ -62,6 +62,7 @@ def main(args=None):
     # Enable logging in relevant packages
     logging.getLogger("kadi").setLevel(opt.log_level)
     fetch.add_logging_handler(level=opt.log_level)
+    fetch.data_source.set("cxc", "maude allow_subset=False")
```

Comment:
Using MAUDE was previously hard-coded into `get_telem_values` which limits its general-purpose utility. Instead we set the data source globally within the validate states app.